pallets/identity/src/migration.rs

```diff
@@ -0,0 +1,28 @@
+use crate::types::{Claim1stKey, Claim2ndKey};
+
+use polymesh_primitives::{CddId, Claim, ClaimType, IdentityClaim};
+use sp_io::hashing::blake2_128;
+
+/// Migrate claim
+pub fn migrate_claim(
+    k1: Claim1stKey,
+    k2: Claim2ndKey,
+    id_claim: IdentityClaim,
+) -> Option<IdentityClaim> {
+    match &k1.claim_type {
+        ClaimType::CustomerDueDiligence => migrate_cdd_claim(&k1, &k2, id_claim),
+        _ => Some(id_claim),
+    }
+}
+
+fn migrate_cdd_claim(
+    k1: &Claim1stKey,
+    _k2: &Claim2ndKey,
```

Comment:
We don't need to pass k2 to this function. In fact, we don't need the full k1 either. We just need `k1.target`.